evidence/edwardmack/0001-edwardmack-09-24-retention-evidence.md

```diff
@@ -0,0 +1,59 @@
+# Evidence-0001: Retention at Rank II
+
+|                 |                                 |
+| --------------- |---------------------------------|
+| **Report Date** | Date of submission (2024/09/25) |
+| **Submitted by**| Edward Mack                     |
+
+
+## Member details
+
+- Matrix username: @mackcom:matrix.org
+- Polkadot address: 1ZYq7qWxvqC1rE4PKnTWwoVJYLtn99p9JM9cUn2WqbWr3Jt
+- Current rank: 2
+- Date of initial induction:
+- Date of last report: NA
+- Area(s) of Expertise/Interest: Gossamer node, parachain subsystems 
+
+
+## Reporting period
+
+- Start date: 2024/07/01
+- End date: 2024/09/25
+
+
+## Evidence
```

Comment:
Please make sure your adhere to the recommendations from the latest template:
"Explain why your contributions in relation to the Polkadot SDK are worthy of retention/promotion. Refer to the terms in Section 6 of the [Manifesto](https://github.com/polkadot-fellows/manifesto/blob/main/manifesto.pdf) and provide links to relevant content (i.e code, articles, media, etc.) to show that you are meeting all the requirements."

This section should consist mainly of a written argument, not a list of PRs, as advised here: https://hackmd.io/@polkadot/guidelines

Reply:
Thank you for the comment and the link highlighting the guidelines, I've updated this.